docs/feature_unicode.md

```diff
@@ -29,6 +29,11 @@ This is the current list of Unicode input method in QMK:
 * UC_WIN: (not recommended) Windows built-in Unicode input. To enable: create registry key under `HKEY_CURRENT_USER\Control Panel\Input Method\EnableHexNumpad` of type `REG_SZ` called `EnableHexNumpad`, set its value to 1, and reboot. This method is not recommended because of reliability and compatibility issue, use WinCompose method below instead.
 * UC_WINC: Windows Unicode input using WinCompose. Requires [WinCompose](https://github.com/samhocevar/wincompose). Works reliably under many (all?) variations of Windows.
 
+To type multiple characters for things like (ノಠ痊ಠ)ノ彡┻━┻, you can use send_unicode_hex_string() much like SEND_STRING() except you would use hex values seperated by spaces.
+For example, the table flip seen above would be send_unicode_hex_string("0028 30CE 0CA0 75CA 0CA0 0029 30CE 5F61 253B 2501 253B")
+
+There are many ways to get a hex code, but an easy one is https://r12a.github.io/app-conversion/ Just make sure to convert to UTF-16 code units, and that is your string.
```

Comment:
Not necessarily. UTF-16 is an encoding, so it produces bytes, not code points. The string used for `send_unicode_hex_string` should be taken from the _Hexadecimal_ section on the site.

For example, the 😍 emoji is code point `1F60D`, whereas its UTF-16 encoding is `D83D DE0D`. The difference doesn't matter for `UNICODE_ENABLE` keymaps, as the highest code point that can be input is `FFFF`, but it might matter for `UNICODEMAP_ENABLE`.

Reply:
Also, make the link a word instead of pasting it raw. For example:

```markdown
There are many ways to get a hex code, but an easy one is through [this site](https://r12a.github.io/app-conversion/).
```